DATABASE.md

```diff
@@ -1,56 +0,0 @@
-### Install
```

Comment:
We no longer need this because the scripts should take care of it for us - and docker will manage versions.